enhancements/kubelet/virtualization-higher-workload-density.md

```diff
@@ -0,0 +1,551 @@
+---
+title: virtualization-higher-workload-density
+authors:
+  - "@enp0s3"
+  - "@iholder101"
+  - "@fabiand"
+reviewers:
+  - "@rphilips"
+  - "@sjenning"
+approvers:
+  - "@mrunalp"
+api-approvers:
+  - "@mrunalp"
+creation-date: "2024-05-05"
+last-updated: "2024-06-12"
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - "https://issues.redhat.com/browse/CNV-28178"
+see-also:
+  - "https://github.com/kubernetes/enhancements/issues/2400"
+status: implementable
+---
+
+# Virtualization higher workload density
+
+## Summary
+
+Fit more workloads onto a given node - achieve a higher workload
+density - by overcommitting it's memory resources. Due to timeline
+needs a two-phased approach is considered.
+
+## Motivation
+
+Today, OpenShift Virtualization is reserving memory (`requests.memory`)
+according to the needs of the virtual machine and it's infrastructure
+(the VM related pod). However, usually an application within the virtual
+machine does not utilize _all_ the memory _all_ the time. Instead,
+only _sometimes_ there are memory spikes within the virtual machine.
+And usually this is also true for the infrastucture part (the pod) of a VM:
+Not all the memory is used all the time. Because of this assumption, in
+the following we are not differentiating between the guest of a VM and the
+infrastructure of a VM, instead we are just speaking colectively of a VM.
```

Comment:
The term "the infrastructure of the VM" might be confusing. 
How about something like:
In the following discussion, we do not differentiate between the VM’s resources and the infrastructure overhead required for the VM.